src/components/carousel/themes/commerce.json

```diff
@@ -74,8 +91,8 @@
 		"styles": {
 			"default": {
 				"border": "none",
```

Comment:
I'm a bit worried about leaving this here. It _should_ work, but it's actually setting `border-top-style: "none"`, `border-right-style: "none"`, etc. I'll have to set up a test tomorrow and check that it behaves as expected.

Reply:
The `border` shorthand in this case should override all the subordinate styles, but testing it would alleviate a concern.  If it did not clear out the logical properties that would be interesting nonetheless.